docusaurus/android_versioned_docs/version-draft/01-basics/01-overview.mdx

```diff
@@ -0,0 +1,140 @@
+
+### UI Components
+
+The UI Components library includes pre-built Android Views to easily load and display data from the Stream Chat API. These include a Channel List and a Message List, a Message Composer View, and more. See the [Getting Started](../02-ui-components/01-getting-started.mdx) page for more details.
+
+This library is built on top of the state library, and offers the quickest integration of Stream Chat into an Android application. It also offers a variety of [theming](../02-ui-components/02-theming.mdx) options to make it fit your app's needs.
+
+You can see the UI Components in action by checking out the [UI Components Sample App](https://github.com/GetStream/stream-chat-android/tree/main/stream-chat-android-ui-components-sample), available in the GitHub repository.
+
+### Compose UI Components
+
+The Compose UI Components library is a chat UI implementation built from scratch with [Jetpack Compose](https://developer.android.com/jetpack/compose). It contains modular Composable functions for building channel lists, messaging screens, and more. See the [Getting Started](../03-compose/01-overview.mdx) page for more details.
+
+This is also built on top of the state library, and offers easy integration of Stream Chat into a Compose-based Android application. It's also highly modular and customizable.
+
+Check out the Compose implementation in action by trying the open-source [Compose UI Components Sample App](https://github.com/GetStream/stream-chat-android/tree/main/stream-chat-android-compose-sample).
+
+:::note
+Using our Compose SDK in an Android Views based app is fully supported. It could provide you an amazing opportunity to explore and adopt Compose.
+:::
+
+### The Right SDK for You
+
+If your use case requires a very high level of customization, runtime theming changes, replacing whole parts of the default UI or stateless components that don't rely on our ViewModels,
+the **Compose SDK is the way to go**. It offers deep customization through several layers - the theme, modifiers for components, bound and stateless component overloads, smaller reusable components and slot APIs.
+
+If, instead, you're looking for a simpler, less customizable SDK which currently offers slightly better performance in some scenarios or your project is limited by technology, then
+the **UI Components SDK is a great option**. It still offers a fair amount of customization through the theme and its attributes and some programmatic ways to customize the UI.
+
+Whichever SDK you choose, **we guarantee you'll be happy and we'll provide amazing support** with your integration, through detailed documentation, guides and direct support.
+
+## Upgrade and Versioning Strategy
+
+Our Android libraries do **not** follow semantic versioning. <!-- TODO Is this accurate? -->
+
+<!-- TODO Is this correct? We increase the minor version for breaking changes? -->
```

Comment:
TODO

Reply:
Correct as per our discussion in sync-up call.

`X.Y.Z`, where **X** is major, **Y** is minor and **Z** is patch versions accordingly.